services/github/github-created-at.service.js

```diff
@@ -0,0 +1,54 @@
+import dayjs from 'dayjs'
+import Joi from 'joi'
+import { age } from '../color-formatters.js'
+import { pathParams } from '../index.js'
+import { formatDate } from '../text-formatters.js'
+import { GithubAuthV3Service } from './github-auth-service.js'
+import { documentation, httpErrorsFor } from './github-helpers.js'
+
+const schema = Joi.object({
+  created_at: Joi.date().required(),
+}).required()
+
+export default class GithubCreatedAt extends GithubAuthV3Service {
+  static category = 'activity'
+  static route = { base: 'github/created-at', pattern: ':user/:repo' }
+  static openApi = {
+    '/github/created-at/{user}/{repo}': {
+      get: {
+        summary: 'Github Created At',
+        description: documentation,
+        parameters: pathParams(
+          {
+            name: 'user',
+            example: 'mashape',
+          },
+          {
+            name: 'repo',
+            example: 'apistatus',
+          },
+        ),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'created at' }
+
+  static render({ createdAt }) {
+    const date = dayjs(createdAt)
+    return {
+      message: formatDate(date),
+      color: age(date),
```

Comment:
This `age` function renders very old repos in red color. I'm not sure if that is good.

Reply:
This helper normally gets used for "last commit" or "last release" badges where recent is generally seen as good. I think for age of the repo we could either just present it in blue (informational) or make an inverse colour scheme for this badge (repos that have been around a long time are "good" whereas repos that have just been created are "bad"). My guess is the reason why you (or another user) wants this badge is to indicate their project has been around a long time, rather than draw attention to the fact they just created it.

Reply:
I think it would be good idea to have an *inversed* color range on `age`. I'll go with adding an `inverse` parameter with a default value of `false` on `age` function initially. That's the only idea I have.

```javascript
function age(date, inverse = false) {
  // ... implementation ...
}
```

If you have another idea, hit me.

Reply:
Completed.